apps/costcontrol/js/costcontrol.js

```diff
@@ -617,8 +617,12 @@ var CostControl = (function() {
       result.status = 'success';
       result.data = lastDataUsage;
 
-      costcontrol.lastDataResults = lastDataUsage;
-
+      // Front-end workaround for Bug 1083680: Noticeable difference between by
```

Comment:
If we move to the [second cache approach](https://github.com/mozilla-b2g/gaia/pull/27347/files#r22933780), we could simplify this comment to:
```
// Once bug 1083680 is solved, both caches should contain the same info so we could use only `costcontrol.lastDataResults`.
```